src/Sprite.cpp

```diff
@@ -462,6 +467,12 @@ void Sprite::UpdateAnimationState()
 // todo: see if "current" practice is just that. -aj
 void Sprite::Update( float fDelta )
 {
+    // fDelta has already been passed in based on the current GameLoop rate,
+    // which might be zero. Use our local timer_ instead.
+	if (!tab_tilde_scaling_enabled_) {
+		fDelta = timer_.GetDeltaTime();
```

Comment:
it seems weird to have some sprites have their own internal timer, and some sprites use `fDelta` passed from the caller. is it possible to standardize on one or the other (perhaps per-sprite timer is too expensive)?

Reply:
We talked offline and this PR will be adjusted in a bit.

Reply:
Yeah I wasn't in love with doing it this way, though fwiw with only the banners having the bool applied the performance hit wasn't too bad. Either way, moot point now with the alternate implementation.